flink-java/src/main/java/org/apache/flink/api/java/RemoteEnvironment.java

```diff
@@ -77,23 +79,6 @@ public RemoteEnvironment(String host, int port, String... jarFiles) {
 		this(host, port, null, jarFiles, null);
 	}
 
-	/**
```

Comment:
I don't think we can remove this public constructor.

Reply:
Reverted.